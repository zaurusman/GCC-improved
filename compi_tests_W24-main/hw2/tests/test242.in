if (0 b)y="awizard" 	   	 	 
			 
 ;if(
false-	 
      		

	
  		
   	 
		
 
	

	j or 	 	7and
 			  
 	(byte) a  	
					  	
  		

		 /6 b + 
   	




 true-

 
 true )a =(  	
int 	  	)true;		
		 

	 		 	
 int
		 

		
	
 				 
	
	z= not
1000000b;
z("h"and  


  
	
 			 	 	 
	
	

	true /false)	


 		 ;//Your   


	laptopis
smallerthenrequired 	 topassthesetests. 	
			  
	

 	 	 	 		  
^^return;xSHy7Bi(

	  	 		
	
	

 
  						

		urAkmmCV7)
	
 
  

		
 
 	
	
 
  	 	
	 		 
	 			 
 ;
bool	 i

	
=false

 
	 

	 - "harry";j
=	
 	
y;//Your grammer
soFATwhen she    
 



 	jumps into    
	
		
the 			  



 
   
	 
  
	

		 	


ocean the

	  			
			
 
 
		
 	
	ocean jumps


	
		 		 	out...
return;
 	 
	y=2;//Your
laptoplookslike a
piazza box.bitch.
 	continue;if 	(729
 )bool EjP33ybM
  	

  ;
{
bytez	  	  	 	


			

		;x =2;//Yourcodeisunreadable. bitch.
}	
//Your partnerisun

	
	 			   

 
 readable...if(
5  
) 
	
	


 
 
 return;

else
a=false;if 		  	  

		

 		
  
		
				
  	
	
	
	 
 	
  
 	  	((bool    		
 	


 

  	
	
 	  
	 	
	
  	
)(true )+379749833583241 or
i(	"you're")-
  	false
>x)if ( 
 
false/	8 *true		 	
  	
		  

	
	

		 
		

	  	
	 		
 

 
 	 	
	
)	intx="you're";  else		

				continue; elsecontinue;    
	
 	
			//Your
grammer isun 			 readable.<3
continue;		 	{y		
(( j(false)

 	 

	 	
	 

 

	 	 )- 	  

 
		("harry")/
	
		 
	   

	"you're" +"harry");		 		
		

continue;bool j



   
			=false;
//Yourlaptophatestheway	 	you call
functions.uwuy(false)	 

;//Yourmamalooks like a piazza box...
}//Yoursexlife shouldgoback

tofirstgrade.:		({ 	
{		

 	
				 
 	
kN3hVE0A1i(
	 	 

			6 b);if( 
"you're" 	 
   )
 
	 
 
  	 			 
  continue;


	 		
 
		
//Your codelooks	  	 

	 	

 	  

 
 	 

 	likea
 


	   
			
	
 	  		   

 

	     	
 	
 	
	piazzabox.<3  	

 	  

		
}
}
		  

	  

 	  					 
	
	if (
		
  
  
	
	
 	


(
	"awizard") -  		 
  		  3
> 
  
	

 
 	 
		falseorz and 

		 

	 
	 
  	 	
 
x		
 	 

			(0b)

 
  		   

	



	
		
    



 	


	  

or4 -true  )
byte 
a=x*true	- true
 


 

 

 	 and 	 
  

			 

			
		   	"a";x=falseand 5b+  	y(743008370688)  +(
 	 	 


	int	 ) 	  
	true- false;//Yoursex lifehatesthe
 	

	
     
	
  	 	
	

   wayyou 	  callfunctions.uwuif( i	)while(z)return;
   	

					 	
 
	  
 
 


 else  



	
	
break	;//Yourfoodmakesrecrutersstart	smokingmother fucker...{return;x(		 
 

 
 

 				 		 
2);//Yourpartnerhatestheway		  
	

		youcall functions...
}

if((	  


 		
				 
byte ) true/"a" )	 			 x(nottrue<(int) 	
 	(i(nottrue	 						 	 ))	
  
  
 
	
     
	 - 	( bool)(  bool) 		  9)	 	

 ;x=j(4 >= 3   
 b)orfalse<=	
2b; if( z 


	 	
){
return;
 
continue;  
		
		
 			 
  

if (""
	

 
 		 
		

 
 	 	
)continue;else	
		if( 
(false 	   )+ 9	
 )
	 
	  


  	 

	 break;elsex(	


	not 2
	b);if(iD1cfZ(2or
true)) 
 				
continue;while  ((		

  	  	

					 	 	 	
int 			) 		
	
	
 	
 
 
		

    XIi5DpkVP + x(iand2
)<=

false)bool y= 
	true ;return;//Your
mama 	
is 




 
 	     	

	unreadable.:)


	
    		
	


	 
  	
 	
		 	 

	 
		

    	while (	
	 
not 	 
	 	 


(false) )		    

  
		  
  				
 
if	

	 (0	
 )	   	bytej  	  	   		;{if(	

 	
 	
	 
	   	y("a wizard")  	
	 
 		 


  

	  		

 andtrue )     		  
 

	   WsEvFy8q9X(7);
if (			
true	)

i(
3 	 		
* true)	;//Your mama	 


	 
	
 
   			 		  
		


	
  
 
 	    					
 
	
isnone existent.uwu
if( 	
	

 	


	
 	
 
 
279936 b		
	
 
		
   
)	 
 		return;//Your sexlife hates thewayyou call functions.uwuwhile (true)
   		   	  
	 		 
	
	bool	i; 
		{b+reak;return 	 ;//Your foodis theonlythingmoreASHCENAZYthenyou.uwubreak;break;}//Yourlaptopis theonly
thingmore
	   		 
ASHCENVAZYthen you. :)	 	  
 	
while 	

(	   
		 
	
 	 	 	   "you 're")-   
 break;//Yourmama			istheonly thing	


 		 
	      moreASHCENAZY then	you.<3? 
	


 		

 		    
 
 

  	
  	  	

bool
	


 	

	

	 	 	 			 	 
		
sN9SyiQGv;	inat z; //Your mamasoFAT3whenshejumpsinto theoce-anthe   oceanjumps Cout.mother  	
 	
	


  

	  
  		
		  
  
  
fucker...

	}
}//Your		 
 ppisthe onlythingmore 
	 ASHCENAZYthen

   		 	 		 	

 
 

you. :)intj=not"you're" 


 		
 	or false;//Your		 
	 	

	 	
   		
	 	ppsoFAT whenshe jumps	 	
  
   	


	

	intothe			  		 
 
 
ocean
  	
  
 
		
 the ocean 	jumpsout. uwu 
{	
 	 	

 	
 
		 
	  
	 
 	  	
	
  		 			

return;	}if 	(  
	 "you're)")
{{ 	return;
 
	 	
} 	x(8)
	;	
 	
    }else		 

		 
	  
  

	

 		
{while
			
		 
(

	 "you're"==false  )return;

 

}
	