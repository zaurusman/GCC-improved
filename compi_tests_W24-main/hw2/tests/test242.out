24: Exp -> NUM B
25: Exp -> STRING
7: Statement -> ID ASSIGN Exp SC
10: Statement -> IF LPAREN Exp RPAREN Statement
2: Statements -> Statement
27: Exp -> FALSE
21: Exp -> ID
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
17: Type -> BYTE
21: Exp -> ID
32: Exp -> LPAREN Type RPAREN Exp
24: Exp -> NUM B
20: Exp -> Exp BINOP Exp
26: Exp -> TRUE
20: Exp -> Exp BINOP Exp
26: Exp -> TRUE
20: Exp -> Exp BINOP Exp
29: Exp -> Exp AND Exp
30: Exp -> Exp OR Exp
16: Type -> INT
26: Exp -> TRUE
32: Exp -> LPAREN Type RPAREN Exp
7: Statement -> ID ASSIGN Exp SC
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
16: Type -> INT
24: Exp -> NUM B
28: Exp -> NOT Exp
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
25: Exp -> STRING
26: Exp -> TRUE
27: Exp -> FALSE
20: Exp -> Exp BINOP Exp
29: Exp -> Exp AND Exp
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
3: Statements -> Statements Statement
line 59: syntax error
